File: Python/Human.py
#Pembuatan class Human yang akan menjadi parent dari SivitasAkademik
class Human:
    
    # Inisialisasi atribut private
    __NIK = ""
    __nama = ""
    __jenis_kelamin = "-"
    
    # Constructor untuk Class Human
    def __init__(self, NIK = "", nama = "", jenis_kelamin = "-"):
        self.__NIK = NIK
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin
    
    
    # Setter dan Getter tiap atribut yang ada pada class Human
    def set_NIK(self, NIK):
        self.__NIK = NIK
    
    def set_nama(self, nama):
        self.__nama = nama
        
    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin
        
    def get_NIK(self):
        return self.__NIK
    
    def get_nama(self):
        return self.__nama
    
    def get_jenis_kelamin(self):
        return self.__jenis_kelamin
    

File: C++/Human.cpp
//Include untuk penggunaan string
#include <iostream>
#include <string>

//Deklarasi std agar tidak perlu menulisnya kembali pada code
using namespace std;

//Membuat class Human
class Human
{
private:

    //Menambahkan attribut yang dibutuhkan
    string NIK;
    string nama;
    char jenis_kelmain;

public:

    //Fungsi construct kosong
    Human(){
        this->NIK = "";
        this->nama = "";
        this->jenis_kelmain = '-';
    }

    //Fungsi construct dengan parameter
    Human(string NIK, string nama, char jenis_kelamin){
        this->NIK = NIK;
        this->nama = nama;
        this->jenis_kelmain - jenis_kelamin;
    }


    //setter dan getter tiap atribut yang ada pada kelas ini
    void set_NIK(string NIK){
        this->NIK = NIK;
    }

    void set_nama(string nama){
        this->nama = nama;
    }

    void set_jenis_kelamin(char jenis_kelamin){
        this->jenis_kelmain = jenis_kelamin;
    }

    string get_NIK(){
        return this->NIK;
    }

    string get_nama(){
        return this->nama;
    }

    char get_jenis_kelamin(){
        return this->jenis_kelmain;
    }

    ~Human(){

    }
};